<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-SQ1HPPK" failures="2" tests="4" name="LoginCheck" time="0.149" errors="0" timestamp="2023-04-08T21:27:07 IST">
  <testcase classname="com.testng.demo.GmailTests" name="gmailInValidLoginCheck" time="0.024">
    <failure type="java.lang.AssertionError" message="Failed intentionally">
      <![CDATA[java.lang.AssertionError: Failed intentionally
at com.testng.demo.GmailTests.gmailInValidLoginCheck(GmailTests.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- gmailInValidLoginCheck -->
  <testcase classname="com.testng.demo.GmailTests" name="gmailValidLoginCheck" time="0.002"/>
  <testcase classname="com.testng.demo.FacebookTests" name="facebookInValidLoginCheck" time="0.003">
    <failure type="java.lang.AssertionError" message="Failed intentionally">
      <![CDATA[java.lang.AssertionError: Failed intentionally
at com.testng.demo.FacebookTests.facebookInValidLoginCheck(FacebookTests.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- facebookInValidLoginCheck -->
  <testcase classname="com.testng.demo.FacebookTests" name="facebookValidLoginCheck" time="0.003"/>
</testsuite> <!-- LoginCheck -->
